src/webportal/src/app/layout/components/alerts.jsx

```diff
@@ -140,7 +140,7 @@ export const NotificationButton = () => {
               <div className={classNames.itemCell} data-is-focusable={true}>
                 {'Issue time: ' + new Date(item.startsAt).toLocaleString()}
                 <br />
-                {'Summary: ' + item.annotations.summary}
+                {item.labels.severity + ':' + item.annotations.summary}
```

Comment:
Can we add color for severity level? Maybe you can refer this: https://hu.pinterest.com/pin/556194622715526398/

Reply:
> Can we add color for severity level? Maybe you can refer this: https://hu.pinterest.com/pin/556194622715526398/

I think the biggest problem of web-portal alerts is that the alerts are not grouped/classified by user / admin. Can we do the colorization after the classification of alerts ? Otherwise it may be confusing to normal users.

Reply:
OK, fine to me. Maybe at first stage we can just sort all alerts by severity level from high to low. I agree we can do it separately